src/class-elements/private-getter-is-not-a-own-property.case

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Caio Lima (Igalia SL). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Private getter is not stored as an own property of objects
+info: |
+  PrivateFieldGet (P, O)
+    1. Assert: P is a Private Name.
+    2. If O is not an object, throw a TypeError exception.
+    3. If P.[[Kind]] is "field",
+      a. Let entry be PrivateFieldFind(P, O).
+      b. If entry is empty, throw a TypeError exception.
+      c. Return entry.[[PrivateFieldValue]].
+    4. Perform ? PrivateBrandCheck(O, P).
+    5. If P.[[Kind]] is "method",
+      a. Return P.[[Value]].
+    6. Else,
+      a. Assert: P.[[Kind]] is "accessor".
+      b. If P does not have a [[Get]] field, throw a TypeError exception.
+      c. Let getter be P.[[Get]].
+      d. Return ? Call(getter, O).
+template: default
+features: [class-methods-private]
+---*/
+
+//- elements
+get #m() { return "Test262"; }
+
+checkPrivateGetter() {
+  assert.sameValue(Object.getOwnPropertyDescriptor(this, "#m"), undefined);
+  assert.sameValue(Reflect.getOwnPropertyDescriptor(this, "#m"), undefined);
+
+  assert.sameValue(this.hasOwnProperty("#m"), false);
+  assert.sameValue(Reflect.has(this, "#m"), false);
+
+  assert.compareArray(Object.getOwnPropertyNames(this), []);
+  assert.compareArray(Reflect.ownKeys(this), []);
+
+  assert.sameValue("#m" in this, false);
+
+  const descriptors = Object.getOwnPropertyDescriptors(this);
+  assert.sameValue("#m" in descriptors, false);
```

Comment:
for the `"#m" in this` I understand you're going through the inheritance chain, but here a hasOwnProperty should do the trick.

Reply:
One way or another, I wonder if should just have a simplification of all these assertions with the following only:

```js
assert.sameValue(this.hasOwnProperty("#m"), false, '#m is not listed as an own property');
assert.sameValue("#m" in this, false, '#m is also not found in the prototype chain');
```

Other than that it seems we are verifying the built-in methods rather than anything else.

Reply:
The main idea here is to cover all users of `[[GetOwnProperty]]` and `[[HasOwnProperty]]`. But maybe only testing own properties and going through the inheritance chain can be enough. WDYT?